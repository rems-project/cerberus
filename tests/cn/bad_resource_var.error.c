void inc(int* p)
/*@ requires take X = RW(p);
             X < 2147483647i32;
    ensures take X2 = RW(p);
            X2 < 2147483647i32; @*/
{
    *p += 1;
}
