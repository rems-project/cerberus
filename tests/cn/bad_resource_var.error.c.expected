return code: 1
[1/1]: inc -- fail
tests/cn/bad_resource_var.error.c:1:1: error: Unprovable constraint
void inc(int* p)
~~~~~^~~~~~~~~~~ 
Constraint from tests/cn/bad_resource_var.error.c:5:13:
            X2 < 2147483647i32; @*/
            ^~~~~~~~~~~~~~~~~~~ 
State file: file:///tmp/state__bad_resource_var.error.c__inc.html
